Source/Turbo.swift

```diff
@@ -19,7 +19,11 @@ public class TurboConfig {
 
     /// Optionally customize the web views used by each Turbo Session.
     /// Ensure you return a new instance each time.
-    public var makeCustomWebView: WebViewBlock = { (configuration: WKWebViewConfiguration) in
+    public var mainWebViewProvider: WebViewBlock = { configuration in
```

Comment:
I prefer to keep the old function and only *add* the two new ones. That gives developers a way to customize both at the same time, like in the Demo app, or dive into more specific customizations.